packages/@atjson/offset-annotations/src/utils/social-urls.ts

```diff
@@ -43,61 +42,6 @@ function getSearchParam(
   return searchParams[name];
```

Comment:
Since this is a breaking change, we could take this opportunity to update the SocialURL interface above like you suggested in the convo here: https://github.com/CondeNast/atjson/pull/381
```
interface SocialURL {
  attributes?: { url: string; height?: string; width?: string }
  AnnotationClass?: typeof IframeEmbed;
}
```